docs/.vuepress/config.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
             "components/frame.md",
             "components/page.md",
             "components/tabview.md",
+            "components/segmentedBar.md",
```

Comment:
can you change the file name according to the other files? it should be segmented-bar.md

Reply:
Inconsistent file name is fixed.